server/sonar-web/src/main/js/apps/projects/filters/__tests__/__snapshots__/Filter-test.tsx.snap

```diff
@@ -37,20 +37,20 @@ exports[`highlights under 1`] = `
           2
         </span>
       </a>
-      <a
-        className="facet search-navigator-facet projects-facet"
-        data-key={3}
-        href="#"
-        key="3"
-        onClick={[Function]}
-      >
-        <span
-          className="facet-name"
-        >
-          3
-        </span>
-      </a>
     </div>
+    <a
```

Comment:
some tests (`hightlights under`, `renders multiple selected` and maybe some else) can be removed